lua/framework/physics/lua_material.cc

```diff
@@ -314,7 +313,7 @@ MatSetProperty(lua_State* L)
 
         const auto file_name = LuaArg<std::string>(L, 4);
 
-        prop->MakeFromOpenSnXSFile(file_name);
+        prop->Initialize(file_name);
       }
       else if (operation_index == static_cast<int>(OpType::EXISTING))
```

Comment:
still confused by exisiting? Does this mean that a XS already exists in memory?

Reply:
Yes, existing means a xs data that's already on the stack.